test/unit/helpers/admin/tabbed_nav_helper_test.rb

```diff
@@ -1,7 +1,8 @@
 require "test_helper"
 
 class TabbedNavHelperTest < ActionView::TestCase
-  test "#secondary_navigation_tabs_items for draft edition edit page" do
+  test "#secondary_navigation_tabs_items for draft edition edit page when user has editor access" do
```

Comment:
Consider adding context blocks to avoid having to append "when" (and "for") to test names (i.e. context blocks for the edition state and the user's access level).

Reply:
yeah, I did consider it, just that I would need to move from using should instead of test